Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10
```

Comment:
Use a specific tag for your base image: Instead of using python:3.10, consider using a specific tag, such as python:3.10-slim or python:3.10-alpine.

python:3.10 image is the official Python image provided by Docker, and it is based on the Debian Buster operating system. This image includes a full Python installation with a variety of standard libraries and tools.

python:3.10-slim image is a smaller version of python:3.10 image and it is optimized for size. It has a smaller image size because it only includes essential Python modules that are necessary to run most Python applications. This image is based on the Debian Buster slim version, which has a smaller footprint than the standard Debian Buster operating system.